app/assets/javascripts/custom.js

```diff
@@ -3,3 +3,32 @@ $(document).bind('page:change', function() {
     CKEDITOR.replace($(this).attr('id'));
   });
 });
+
+function call_ajax() {
+  let name = $('#find_by_name').val()
```

Comment:
```suggestion
  let name = $('#find_by_name').val();
```
thêm tương tự khi kết thúc hàm